src/java.base/share/classes/javax/crypto/KEM.java

```diff
@@ -65,24 +65,40 @@
  * new shared secret and key encapsulation message.
  * <p>
  *
- * Example:
+ * Example operation using a ficticious {@code KEM} algorithm {@code ABC}:
  * {@snippet lang = java:
- *    // Receiver side
- *    var kpg = KeyPairGenerator.getInstance("X25519");
- *    var kp = kpg.generateKeyPair();
+ *     // Receiver side
+ *     KeyPairGenerator g = KeyPairGenerator.getInstance("ABC");
+ *     KeyPair kp = g.generateKeyPair();
+ *     publishKey(kp.getPublic());
  *
- *    // Sender side
- *    var kem1 = KEM.getInstance("DHKEM");
- *    var sender = kem1.newEncapsulator(kp.getPublic());
- *    var encapsulated = sender.encapsulate();
- *    var k1 = encapsulated.key();
+ *     // Sender side
+ *     KEM senderKEM = KEM.getInstance("ABC-KEM");
+ *     PublicKey receiverPublicKey = retrieveKey();
+ *     ABCKEMParameterSpec spec = new ABCKEMParameterSpec(...);
```

Comment:
Maybe we can rename `spec` to `senderSpec`. We already have `senderKEM` and `senderSecret`.

Reply:
Sure, I debated on this...will update.